system-uptime-tracking/xml/art_system-uptime-tracking.xml

```diff
@@ -0,0 +1,375 @@
+          <filename>/etc/zypp/suse-uptime.log</filename>. These log files are
+          uploaded to SCC/RMT by the
+          <systemitem class="daemon">suseconnect-keepalive</systemitem> service,
+          which is triggered by the
+          <systemitem class="daemon">suseconnect-keepalive</systemitem> timer
+          every 24 hours. Verify that the timer is running.
+        </para>
+<screen>&prompt.user;systemctl status suseconnect-keepalive.timer</screen>
+      </step>
+    </procedure>
+  </sect1>
+  <sect1 xml:id="uptime-data-retrieve">
+    <title>Retrieving the uptime tracker data</title>
+
+    <para>
+      You can retrieve the data that the uptime tracker collects from SCC and
+      export them to the CSV format via its REST API.
+    </para>
+
+    <procedure>
+      <step>
+        <para>
+          Obtain your organization mirroring credentials from the
+          <guimenu>Proxies</guimenu> tab in the SCC Web at
+          <link
+          xlink:href="https://scc.suse.com"/>.
+        </para>
+      </step>
+      <step>
+        <para>
+          Get the list of all systems for your organization.
+        </para>
+<screen>
+&prompt.user;curl -X 'GET' \
+-u'<replaceable>USERNAME</replaceable>:<replaceable>PASSWORD</replaceable>' \
+'https://scc.suse.com/connect/organizations/systems' \
+-H 'accept: application/json'
+</screen>
+        <para>
+          In the list of systems, identify the client for which you want to
+          retrieve the uptime data by matching its credentials found in the
+          client's <filename>/etc/zypp/credentials.d/SCCcredentials</filename>
+          file.
+        </para>
+      </step>
+      <step>
+        <para>
+          Retrieve the uptime data. The following examples retrieve uptime data
+          for the system with ID <literal>16300735</literal> from Apr 24th, 2024
+          to May 1st, 2024. You can use either the <command>curl</command> or
+          <command>wget</command> commands for the retrieval.
+        </para>
+        <substeps>
+          <step>
+            <para>
+              To retrieve the data using the <command>curl</command> command,
+              run the following:
+            </para>
+<screen>
+&prompt.user;curl -X 'GET' \
+-u'<replaceable>USERNAME</replaceable>:<replaceable>PASSWORD</replaceable>' \
+'https://scc.suse.com/connect/organizations/system_uptimes?SystemId=16300735&amp;OnlineFrom=2024-04-24&amp;OnlineTo=2024-05-01' \
+-H 'accept: text/csv'
+</screen>
+          </step>
+          <step>
+            <para>
+              To retrieve the data using the <command>wget</command> command,
+              run the following:
+            </para>
+<screen>
+&prompt.user;wget --no-check-certificate \
+--user=<replaceable>USERNAME</replaceable> --password=<replaceable>PASSWORD</replaceable> \
+--header='accept: text/csv' \
+'https://scc.suse.com/connect/organizations/system_uptimes'
```

Comment:
the curl command includes params (systemid, online from, online to), but the wget command doesn't. is that intentional?

Reply:
it is not intentional. Updated the [https://confluence.suse.com/display/CloudSolutions/White+paper+documentation](url) section for query parameter "Here is the example of wget command retrieves system uptime data for system 16300735"

Reply:
Updated the document to match curl query parameter date and system id
`wget --user=<username> --password=<Password>'[http://192.168.196.107:3000/connect/organizations/system_uptimes?SystemId=16300735&OnlineFrom=2024-04-24&OnlineTo=2024-05-01'](http://192.168.196.107:3000/connect/organizations/system_uptimes?SystemId=16300735&OnlineFrom=2024-04-24&OnlineTo=2024-05-01%27)`